src/components/header/color-panel/color-panel.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import classnames from 'classnames';
+
+import { Styles, ColorsStyles } from './color-panel.style';
+import { Colors } from '../../../constants';
+
+export const ColorPanel: FC = () => {
```

Comment:
I think that its panel has more than 1 color. But when I read this name I don`t think so